### DISCUSSION TITLE
In which version of Material UI v6 do you plan to migrate the entire MRT v2?

### DISCUSSION THREAD

**Discussion by MarekPatyna** [KevinVandy/material-react-table#1177] [General]
Currently, the Material UI is at version 6.0.0-alpha.14. Do you plan to migrate the entire MRT v2 only when the official version 6.0 without the alpha or beta marker comes out, or are plans different though?

@KevinVandy

**Comment by KevinVandy** [comment]
It's more of a situation that I don't have time for the migration until September. My goal is to move totally to all of Pigment CSS, MUI v6, and TanStack Table v9 (which I'm currently working on).

**Comment by MarekPatyna** [comment]
Thank you for your reply and good luck with the migration.
